routes/lotto.js

```diff
@@ -0,0 +1,20 @@
+var express = require('express');
+var router = express.Router();
+const cors = require('cors');
+const random = require('../modules/randomPicks');
+
+/** 랜덤 로또 번호 생성  */
+router.post('/randomPicks', cors(), (req, res) => {
+  var fixed = [], excluded = [];
+  if(req.body.fixed !== undefined) fixed = req.body.fixed;
+  if(req.body.excluded !== undefined) excluded = req.body.excluded;
```

Comment:
```suggestion
  const fixed = req?.body?.fixed ?? [];
  const excluded = req?.body?.excluded ?? [];
```
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Optional_chaining 참조